views/login.handlebars

```diff
@@ -0,0 +1,31 @@
+<!-- TO BE REPLACED WITH BUTTON -->
+<button id="modalBtn" class="logInButton">LOG IN</button>
+
+<div id="infoModal" class="modal">
+  <div class="modalContent">
+    <div class="modalHeader">
+      <span id="closeBtn">&#x2718;</span>
+      <h2>Registration</h2>
+    </div>
+    <div class="modalBody">
+      <form id="info">
+        <label for="email">What is your email?</label><br />
+        <input
+          type="email"
+          id="email"
+          name="email"
+          value="Supergoodgamer@email.com"
+        /><br />
+        <label for="password">What would you like to make your password?(MIN RQS
+          HERE)</label><br />
+        <input type="password" id="password" name="password" required /><br />
+
+        <button type="submit" class="submitModal" id="submit">
+          LETS GO!
+        </button>
+      </form>
+    </div>
+  </div>
+</div>
+
+<script src="/js/modal.js"></script>
```

Comment:
It's best practice to leave a empty line at the end of a file. This is for formatting issues that can arise when automated scripts merge files and such, although that's not likely to happen with us, Github has annoying reminders.